### PR TITLE
Add operation result objects

### PR DESCRIPTION
**Pull request by cednation**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Comment by adxsdknet** [comment]
# Automation for azure-sdk-for-net 
A PR has been created for you: 
https://github.com/azure/azure-sdk-for-net/pull/5579 
.NET SDK Commits: 
https://github.com/adxsdknet/azure-sdk-for-net/pull/5579/commits/6104aef65851505bbd3a06db791b20e31dd2a92b

File: specification/azsadmin/resource-manager/fabric/Microsoft.Fabric.Admin/preview/2016-05-01/ApplicationOperationResults.json [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+        "ApplicationOperationResultModel": {
+            "description": "Application operation result properties.",
+            "type": "object",
+            "properties": {
+                "provisioningState": {
+                    "description": "Success or failure of operation.",
+                    "type": "string"
+                },
+                "error": {
+                    "code": "Programatic error code.",
```
Comment (nschonni) [inline specification/azsadmin/resource-manager/fabric/Microsoft.Fabric.Admin/preview/2016-05-01/ApplicationOperationResults.json]: Programatic -> Programmatic

File: specification/azsadmin/resource-manager/fabric/Microsoft.Fabric.Admin/preview/2016-05-01/ComputeOperationResults.json [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+        "ComputeOperationResultModel": {
+            "description": "Compute operation result properties.",
+            "type": "object",
+            "properties": {
+                "provisioningState": {
+                    "description": "Success or failure of operation.",
+                    "type": "string"
+                },
+                "error": {
+                    "code": "Programatic error code.",
```
Comment (nschonni) [inline specification/azsadmin/resource-manager/fabric/Microsoft.Fabric.Admin/preview/2016-05-01/ComputeOperationResults.json]: Programatic -> Programmatic

File: specification/azsadmin/resource-manager/fabric/Microsoft.Fabric.Admin/preview/2016-05-01/NetworkOperationResults.json [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+        "NetworkOperationResultModel": {
+            "description": "Network operation result properties.",
+            "type": "object",
+            "properties": {
+                "provisioningState": {
+                    "description": "Success or failure of operation.",
+                    "type": "string"
+                },
+                "error": {
+                    "code": "Programatic error code.",
```
Comment (nschonni) [inline specification/azsadmin/resource-manager/fabric/Microsoft.Fabric.Admin/preview/2016-05-01/NetworkOperationResults.json]: Programatic -> Programmatic

File: specification/azsadmin/resource-manager/fabric/Microsoft.Fabric.Admin/preview/2016-05-01/StorageOperationResults.json [hunk 4]
```diff
@@ -0,0 +1,185 @@
...
+        "StorageOperationResultModel": {
+            "description": "Storage operation result properties.",
+            "type": "object",
+            "properties": {
+                "provisioningState": {
+                    "description": "Success or failure of operation.",
+                    "type": "string"
+                },
+                "error": {
+                    "code": "Programatic error code.",
```
Comment (nschonni) [inline specification/azsadmin/resource-manager/fabric/Microsoft.Fabric.Admin/preview/2016-05-01/StorageOperationResults.json]: Programatic -> Programmatic

File: specification/azsadmin/resource-manager/fabric/Microsoft.Fabric.Admin/preview/2016-05-01/ComputeOperationResults.json (hunk 2 above)
Comment (cednation) [reply specification/azsadmin/resource-manager/fabric/Microsoft.Fabric.Admin/preview/2016-05-01/ComputeOperationResults.json]: Thanks, fixed.

File: specification/azsadmin/resource-manager/fabric/Microsoft.Fabric.Admin/preview/2016-05-01/StorageOperationResults.json (hunk 4 above)
Comment (cednation) [reply specification/azsadmin/resource-manager/fabric/Microsoft.Fabric.Admin/preview/2016-05-01/StorageOperationResults.json]: Thanks, fixed.

File: specification/azsadmin/resource-manager/fabric/Microsoft.Fabric.Admin/preview/2016-05-01/NetworkOperationResults.json (hunk 3 above)
Comment (cednation) [reply specification/azsadmin/resource-manager/fabric/Microsoft.Fabric.Admin/preview/2016-05-01/NetworkOperationResults.json]: Thanks, fixed.

**Comment by bganapa** [comment]
@martabarillas, @knithinc to review

File: specification/azsadmin/resource-manager/fabric/Microsoft.Fabric.Admin/preview/2016-05-01/ApplicationOperationResults.json [hunk 5]
```diff
@@ -0,0 +1,188 @@
+{
+    "swagger": "2.0",
```
Comment (sergey-shandar) [inline specification/azsadmin/resource-manager/fabric/Microsoft.Fabric.Admin/preview/2016-05-01/ApplicationOperationResults.json]: New swagger files has to be referenced from `readme.md`

**Review by sergey-shandar — CHANGES_REQUESTED** [review]

File: specification/azsadmin/resource-manager/fabric/Microsoft.Fabric.Admin/preview/2016-05-01/ApplicationOperationResults.json (hunk 5 above)
Comment (cednation) [reply specification/azsadmin/resource-manager/fabric/Microsoft.Fabric.Admin/preview/2016-05-01/ApplicationOperationResults.json]: Done

File: specification/azsadmin/resource-manager/fabric/Microsoft.Fabric.Admin/preview/2016-05-01/ApplicationOperationResults.json [hunk 6]
```diff
@@ -0,0 +1,188 @@
...
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "OK",
+                        "schema": {
+                            "$ref": "#/definitions/ApplicationOperationResult"
+                        }
+                    },
+                    "404": {
```
Comment (bganapa) [inline specification/azsadmin/resource-manager/fabric/Microsoft.Fabric.Admin/preview/2016-05-01/ApplicationOperationResults.json]: @sergey-shandar  Do you recommend having 404 error codes in swagger files? or they need to be removed? I believe if we keep the 404s here, the SDKs wont throw exception and silently eat the exception..

Comment (bganapa) [reply specification/azsadmin/resource-manager/fabric/Microsoft.Fabric.Admin/preview/2016-05-01/ApplicationOperationResults.json]: I think its fine, we include 404s in all our admin swagger files..

**Review by bganapa — APPROVED** [review]

**Comment by bganapa** [comment]
@sergey-shandar This should be good to merge...

**Comment by KrisBash** [comment]
This appears to be a new RP. Have you done a review with armapireview? If not, please schedule one to proceed.

**Comment by bganapa** [comment]
@KrisBash This is azure stack admin API surface, specific to AZURESTACK. These have gone thru the AZURESTACK api review board. We have an understanding with Gaurav that these wont go thru ARM api Review board.

**Comment by bganapa** [comment]
@sergey-shandar This should be good to merge.. Are you waiting on something?

**Review by sergey-shandar — APPROVED** [review]

**Pull request merged**